pkg/genericapiserver/genericapiserver.go

```diff
@@ -698,6 +716,20 @@ func (s *GenericAPIServer) installAPIGroup(apiGroupInfo *APIGroupInfo) error {
 	return nil
 }
 
+func (s *GenericAPIServer) GetCIDRToAddressMap() []unversioned.CIDRToAddress {
+	return []unversioned.CIDRToAddress{
+		{
+			CIDR:    s.ServiceClusterIPRange.String(),
```

Comment:
is this right?  The ServiceClusterIPRange is the range of addresses assigned to Services